sound/soc/sof/ipc.c

```diff
@@ -532,7 +532,7 @@ static void ipc_stream_message(struct snd_sof_dev *sdev, u32 msg_cmd)
 		ipc_xrun(sdev, msg_id);
 		break;
 	default:
-		dev_err(sdev->dev, "error: unhandled stream message %x\n",
+		dev_err(sdev->dev, "error: unhandled stream message 0x%x\n",
```

Comment:
I meant "#%x" as documented e.g. in https://man7.org/linux/man-pages/man3/printf.3.html

Reply:
you mean: "%#x", right?

Since I will anyway going to update the PR, I'll make this change as well.

Reply:
But do we care about coding style and consistency?
```
git grep -nc "%#x" sound/soc/sof/
sound/soc/sof/debug.c:1
sound/soc/sof/ops.h:2
sound/soc/sof/topology.c:1
```
vs
```
git grep -c -e "0x%x" -e "0x%08x" sound/soc/sof/
sound/soc/sof/control.c:2
sound/soc/sof/imx/imx-common.c:1
sound/soc/sof/imx/imx8.c:2
sound/soc/sof/imx/imx8m.c:2
sound/soc/sof/intel/atom.c:1
sound/soc/sof/intel/bdw.c:5
sound/soc/sof/intel/byt.c:4
sound/soc/sof/intel/cnl.c:2
sound/soc/sof/intel/hda-ctrl.c:8
sound/soc/sof/intel/hda-dsp.c:4
sound/soc/sof/intel/hda-ipc.c:3
sound/soc/sof/intel/hda-stream.c:5
sound/soc/sof/intel/hda.c:2
sound/soc/sof/intel/pci-tng.c:4
sound/soc/sof/ipc.c:11
sound/soc/sof/loader.c:15
sound/soc/sof/pcm.c:2
sound/soc/sof/topology.c:7
sound/soc/sof/xtensa/core.c:1
```
The "0x%x" is more like default in sof.

Reply:
not really @ujfalusi, it was more ignorance that %# even existed rather than a conscious decision to use 0x%x...
I think we should gradually convert all those 0x%x to the recommended notation. all the last patches use 0x%#.

Reply:
@plbossart, OK. I'll update the PR in the morning.